README.md

```diff
@@ -25,7 +25,7 @@
 - ~~Loader~~
 - ~~Toast~~
 - Banner
-- Tabs
+- Tabs (in progress)
```

Comment:
Might wanna just cross it out since we're nearing the finish line 😄

Reply:
Doh! Yeah, I suppose I should have stared with it crossed off. 🤣